three.js/src/threex/threex-artoolkitcontext-nft.js

```diff
@@ -29,13 +29,18 @@ ARjs.Context = THREEx.ArToolkitContext = function (parameters) {
         // the patternRatio inside the artoolkit marker - artoolkit only
         patternRatio: 0.5,
 
+        // Labeling mode for markers - ['black_region', 'white_region']
```

Comment:
we don't need this and also other part of the code, but i'm not sure if we can remove them. Maybe better to do in another PR.

Reply:
we can do also on this PR but before we have to reach a stable state where everything works, marker and nft in aframe and three.js.

we have to remember about this.